src/index.js

```diff
@@ -46,6 +46,7 @@ class ReactIntlUniversal {
       cookieLocaleKey: null, // Cookie's Key to determine locale. Example: if cookie=lang:en-US, then set it 'lang'
       locales: {}, // app locale data like {"en-US":{"key1":"value1"},"zh-CN":{"key1":"值1"}}
       warningHandler: console.warn, // ability to accumulate missing messages using third party services like Sentry
+      escapeHtml: true,
```

Comment:
If escapeHtml is true, then any escapable html strings will be escaped in the get/getHtml calls.